pkg/asset/machines/azure/azuremachines.go

```diff
@@ -293,3 +329,16 @@ func CapzTagsFromUserTags(clusterID string, usertags map[string]string) (capz.Ta
 	}
 	return tags, nil
 }
+
+// Storage account names can't be more than 24 characters.
+func getStorageAccountName(infraID string) string {
```

Comment:
I guess this is a copy from the same function `pkg/infrastructure/azure/azure.go`? 

Maybe we can add a comment to reference that function so that if we ever have to change, we make sure to change both?

Reply:
moved it to pkg/types and exported, so both now use that.